.npmrc

```diff
@@ -1 +0,0 @@
-//registry.npmjs.org/:_authToken=${NPM_TOKEN}
```

Comment:
I hope you don't mind, I just use ~/.npmrc and this makes `npm install` and other things fail if env var is not set.